src/data/constants.js

```diff
@@ -1,11 +1,21 @@
 const VALIDATION_STATUS = {
```

Comment:
Change this to an Object { key: VALIDATION_STATUS, label: VALIDATION_STATUS_LABEL }